.github/workflows/ci.yml

```diff
@@ -21,36 +18,27 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - uses: ruby/setup-ruby@v1
-
-    - name: Install PostgreSQL 11 client
-      run: sudo apt-get -yqq install libpq-dev
-
     - uses: actions/cache@v1
       with:
         path: vendor/bundle
-        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
-        restore-keys: |
-          ${{ runner.os }}-gems-
+        key: bundle-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: bundle
 
-    - name: Build App
+    - run: sudo apt-get -yqq install libpq-dev
```

Comment:
Interesting, why move this above the env?

Reply:
To keep the `uses` together.